scripts/bd_to_prod.sh

```diff
@@ -32,15 +32,15 @@ echo "Creating layer file"
 ./scripts/prepare_layer_files.sh
 
 echo "Creating lumigo-node layer"
-../utils/common_bash/create_layer.sh lumigo-node-tracer ALL nodejs "nodejs10.x nodejs8.10"
+../utils/common_bash/create_layer.sh lumigo-node-tracer ALL nodejs "nodejs10.x node12.x"
```

Comment:
```suggestion
../utils/common_bash/create_layer.sh lumigo-node-tracer ALL nodejs "nodejs10.x nodejs12.x"
```
It was with `js` before and it is also with `js` in other places.